lib/components/programs-list/index.tsx

```diff
@@ -73,7 +34,9 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
       />
       <Flex flexDirection="column" alignItems="start">
         <Heading>{title}</Heading>
-        <Text fontSize="lg">{description}</Text>
+        <Text fontSize="lg" whiteSpace="pre-line">
```

Comment:
Why are we adding the `whiteSpace="pre-line"` attribute? This replaces white space with more than 2 spaces with a single space. Is our content just not formatted correctly?